tests/integration/commands.py

```diff
@@ -24,6 +24,7 @@ class Commands:
 # This is matched with E2E test run of github-runner-operator charm.
 TEST_RUNNER_COMMANDS = (
     Commands(name="simple hello world", command="echo hello world"),
+    Commands(name="print groups", command="groups"),
```

Comment:
Does this check make sure that the user is part of the sudoers group?

Reply:
Updated the checks to grep for sudo in groups, which should exit with non-zero return code. Thank you!